src/include/migraphx/half.hpp

```diff
@@ -28,11 +28,12 @@
 #include <half/half.hpp>
 #include <migraphx/config.hpp>
 #include <migraphx/float8.hpp>
+#include <migraphx/generic_float.hpp>
```

Comment:
What does the `ifdef` on L45 do? If that is removed, could we remove the `<half/half.hpp>` include here?

Reply:
Yea I think we should remove `<half/half.hpp>` includes everywhere from migraphx, and we can remove the ifdef from below.